snuba/replacers/errors_replacer.py

```diff
@@ -203,8 +206,13 @@ def process_message(
             raise InvalidMessageType("Invalid message type: {}".format(type_))
 
         if processed is not None:
-            bypass_projects = get_config("replacements_bypass_projects", "[]")
-            projects = json.loads(cast(str, bypass_projects))
+            manual_bypass_projects = get_config("replacements_bypass_projects", "[]")
+            auto_bypass_projects = list(
+                get_config_auto_replacements_bypass_projects(datetime.now())
+            )
+            if manual_bypass_projects is not None:
+                auto_bypass_projects.extend(manual_bypass_projects)
+            projects = json.loads(cast(str, auto_bypass_projects))
```

Comment:
I made an update to this part of the code - I didn't realize `replacements_bypass_projects` is passed in as a string even though it's a list, and I cleaned it up a little more.

> what happens if someone enters invalid json into the runtime config?
Good point - ideally the safety checks for the manual `replacements_bypass_projects` should've been included in its original PR, so I included a try block for now. This PR intends to use the existing logic for manual projects retrieval, so I think if we want to add validity checks during manual projects retrieval, I should do it in a separate PR. 

> also isn't auto_bypass_projects already a list? Do you mean manual_bypass_projects?
No, we need to combine auto and manual projects because all of them need to be skipped. I agree the naming before was a little confusing though, so I made the it clearer.